### PR TITLE
Add `wasm32` as a cross-compile target

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -147,6 +147,7 @@ jobs:
           - mipsel-unknown-linux-gnu
           - powerpc-unknown-linux-gnu
           - powerpc64-unknown-linux-gnu
+          - wasm32-unknown-unknown
```
Comment (mxinden) [inline .github/workflows/rust.yml:150]:
```suggestion
          - wasm32-unknown-unknown
          - wasm32-unknown-emscripten,
          - wasm32-wasi
```

Would it make sense to test against these targets as well?

**Review by mxinden — COMMENTED** [review]
:pray: this help is much appreciated!

File: .github/workflows/rust.yml (hunk 1 above)
Comment (thomaseizinger) [reply .github/workflows/rust.yml:150]: I think `wasm32-unknown-unknown` is the most restrictive one. If it works for those, it should work for the others too :)

Comment (mxinden) [reply .github/workflows/rust.yml:150]: Understood. Let's proceed as is then.

**Review by mxinden — APPROVED** [review]

**Pull request merged**